x-pack/test/api_integration/apis/management/index_management/indices.js

```diff
@@ -203,7 +210,8 @@ export default function ({ getService }) {
         // We need to sort the keys before comparing then, because race conditions
         // can cause enrichers to register in non-deterministic order.
         const sortedExpectedKeys = expectedKeys.sort();
-        const sortedReceivedKeys = Object.keys(body[0]).sort();
+        const sortedReceivedKeys = Object.keys(indexCreated).sort();
```

Comment:
I'm hoping that by asserting against a specific index this test will be more stable. It's possible for an index to also have a `date_stream` property (which is not included in the `expectedKeys`), so this may have been occasionally failing because the index return from `body[0]` had a date stream associated with. Related comment: https://github.com/elastic/kibana/issues/64473#issuecomment-775322843.